README.md

```diff
@@ -33,6 +33,9 @@ WahWah support ruby 2.5+
 WahWah is so easy to use.
 
 ```ruby
+require 'wahwah'
+require 'stringio'
```

Comment:
Cause already require 'stringio' in lib/wahwah.rb, don't need require 'stringio'  manually here. should delete this `require 'stringio'`  line in README

Reply:
oops, I didn't keep that when I reverted the previous commit! 🙈